data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/impl/ReceiverVerticle.java

```diff
@@ -82,6 +81,18 @@ public class ReceiverVerticle extends AbstractVerticle implements Handler<HttpSe
     private MessageConsumer<Object> messageConsumer;
     private IngressProducerReconcilableStore ingressProducerStore;
 
+    private SecretWatcher secretWatcher;
+
+    public void setSecretVolumePath(String path) {
+        this.SECRET_VOLUME_PATH = path;
+        this.TLS_KEY_FILE_PATH = SECRET_VOLUME_PATH + "/tls.key";
+        this.TLS_CRT_FILE_PATH = SECRET_VOLUME_PATH + "/tls.crt";
+    }
```

Comment:
Can we do this injection as part of the constructor?

Also, I'd make `SECRET_VOLUME_PATH`, `TLS_KEY_FILE_PATH`,  `TLS_CRT_FILE_PATH` `final` instance variables instead of being static and non final